source/content/global-cdn.md

```diff
@@ -32,6 +32,24 @@ We recommend installing the Pantheon Advanced Page Cache [plugin](https://wordpr
 
 For more details, see [Clearing Caches for Drupal and WordPress](/clear-caches).
 
+## Persistent Cache
```

Comment:
FWIW, I object to the name that the company is converging on for the feature because it implies a specific behavior by the CDN - storing objects with persistence guarantees - that Fastly explicitly disclaims and so we aren't delivering.

I think there are many great ways we could name this feature that convey the customer value without using techy words like "cache," thereby sidestepping potential deception. For example "Enhanced Uptime Protection" or something like that.